docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  uajp:
+    build: .
+    tty: true
+    depends_on:
+      - mysql_db
+  
+  mysql_db:
+    image: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+    #volumes:  
```

Comment:
i recommend setting Docker volume for data persistence.

https://docs.docker.com/compose/compose-file/#volumes